lib/pytis/util/test.py

```
@@ -330,20 +276,63 @@ def squarer(key):
+    # Test public API.
+    @pytest.fixture(scope="class")
+    def resolver(self):
         from .resolver import Resolver
+        return Resolver(search=('pytis.defs', 'pytis.defs.profiles'))
+
+    def test_get(self, resolver):
         import pytis.presentation
-        r = Resolver(search=('pytis.defs', 'pytis.defs.profiles'))
-        view = r.get('help.Help', 'view_spec')
+        view = resolver.get('help.Help', 'view_spec')
         assert isinstance(view, pytis.presentation.ViewSpec)
```

Ruff (0.12.2)

Use of `assert` detected

(S101)